src/main/java/core/basesyntax/data/dao/FileReaderImpl.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.data.dao;
+
+import core.basesyntax.data.model.FruitTransaction;
+import core.basesyntax.data.model.OperationType;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    private static final String INPUT_DATA = "fruit_shop_transactions.csv";
+
+    @Override
+    public List<FruitTransaction> readFile(String fileName) {
+        List<FruitTransaction> transactions = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new java.io.FileReader(fileName))) {
```

Comment:
Ensure that the file path provided to the `FileReader` is correct and accessible. The hardcoded path in `INPUT_DATA` is not used in this method, which might lead to confusion.